packages/smb-onboarder/src/components/ui/negotiators-list/negotiators-list.tsx

```diff
@@ -166,8 +174,8 @@ export const getDataTable = ({
       { value: _eTag, className: 'hidden' },
     ]
   })
-  const dataTable = [headers, ...dataRows]
-  return dataTable
+
+  return headers.length ? [headers, ...dataRows] : dataRows
```

Comment:
What does `headers.length` mean in this case? I think it should be `headers.length > 0` or something like that

Reply:
It's the same